rust/noosphere-ns/src/bin/orb-ns/runner/config.rs

```diff
@@ -65,22 +60,32 @@ impl RunnerConfig {
             CLICommand::Run {
                 config,
                 key,
+                mut bootstrap,
                 mut port,
             } => match config {
                 Some(config_path) => {
                     let toml_str = tokio::fs::read_to_string(&config_path).await?;
-                    let config: CLIConfig = toml::from_str(&toml_str)?;
+                    let config: CLIConfigFile = toml::from_str(&toml_str)?;
```

Comment:
Will the NS be able to be initialized before a configuration file is present?

Reply:
The CLI flags support defining a listening address and bootstrap peers, though limited to one node -- the config file has additional configurations possible, namely DHTConfig values (e.g. libp2p timeouts and knobs) and multiple nodes per parent process (unclear how useful multiple nodes are on a single machine, though more of a stopgap until nodes are e.g. across the globe). DHTConfig values can be mapped to CLI flags of course if it looks to be valuable. Is that the scenario you're thinking of? (the "before" is confusing me a bit)